dashboard/db/migrate/20160802170542_add_readonly_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddReadonlyUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
+  end
+end
```

Comment:
They can be nil. In my local db it looks like view_answers defaults to NULL everywhere, which gets treated as false afaict, which is the desired behavior. Would it be better to just set to 0 (false) everywhere? If so, what's the syntax to do that as part of the migration?